app/models/note.rb

```diff
@@ -0,0 +1,17 @@
+# == Schema Information
+#
+# Table name: notes
+#
+#  id         :bigint(8)        not null, primary key
+#  title      :string           not null
+#  content    :string           not null
+#  note_type  :string           not null
+#  user_id    :bigint(8)
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Note < ApplicationRecord
+  belongs_to :user
+  has_one :utility, through: :user
+  validates :user_id, :title, :content, :note_type, presence: true
```

Comment:
Muy bien!! Aún nos falta una validación. Debería poder crear una nota indicando cualquier `note_type`? Voy dejando un enlace que te va a resultar útil: https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails/